app/utils/IncomeSourcesUtils.scala

```diff
@@ -76,7 +108,7 @@ object IncomeSourcesUtils {
     val skipAccountingMethod: Boolean = userActiveBusinesses.isEmpty
 
     val result: Option[Either[Throwable, CheckBusinessDetailsViewModel]] = for {
-      businessName <- user.session.data.get(businessName)
+      businessName <- user.session.data.get(businessName) //sessionService.get(businessName)
```

Comment:
not sure what we have changed in the line?

Reply:
Have removed comment